src/nupic/swarming/permutations_runner.py

```diff
@@ -367,7 +367,10 @@ def _setUpExports(exports):
     return ret
   exportDict = json.loads(exports)
   for key in exportDict.keys():
-    ret+= "export %s=%s;" % (str(key), str(exportDict[key]))
+    if (sys.platform.startswith('win')):
```

Comment:
Yuck, we really need to rewrite this code to properly set up the environment rather than passing on command line. This is fine for this PR though.